fastlane/Fastfile

```diff
@@ -59,14 +65,16 @@ platform :ios do
     rescue
       fail "please add a CHANGELOG.latest.md file before calling this lane"
     end
+    commit_hash = last_git_commit[:commit_hash]
```

Comment:
would this return the last commit in the repository? Or is this equivalent to HEAD?

Reply:
from the [source code](https://github.com/fastlane/fastlane/blob/e874a47c6e2e0e61590a03d3b71e75e5a505d1ce/fastlane/lib/fastlane/helper/git_helper.rb#L76), this is running `git log -1`, so it should be equivalent to head